src/app/pages/website/modules/modules.component.html

```diff
@@ -0,0 +1,51 @@
+<littil-title>Modules</littil-title>
+
+<littil-content-container>
+    <div class="grid grid-cols-12 gap-x-4 my-12">
+        <article class="col-span-12 md:col-span-8">
+            <p>Project LITTIL is ontstaan uit een idee van Stichting Devoxx4Kids Nederland. 
+                Deze stichting heeft meerdere modules in haar beheer waarmee vrijwilligers 
+                programmeerlessen kunnen geven aan kinderen in de leeftijd van 8 tot 14 jaar.
+            </p>
+            <div class="flex flex-col gap-y-4 mt-6">
+                <h3>Indien je aan de slag wilt met een les module, dan raden wij je aan om de 
```

Comment:
Algemene opmerking: Headings (h1, h2, etc) kunnen beter niet overgeslagen worden. Dus gebruik eerst \<h2\>, en als daar een niveau onder zit, gebruik daarvoor dan \<h3\>.

Echter, in dit geval is een header wellicht niet gewenst. Headers worden gebruikt om te navigeren op een pagina (bijvoorbeeld door screenreaders). Dus headers dienen korte kopjes te zijn die omschrijven waar een sectie over gaat.

Indien deze tekst puur \<h3\> is voor styling (het dikgedrukte), gebruik dan \<b\>. Alternatief is \<strong\>, die het ook visueel dikgedrukt maakt, maar semantisch aangeeft dat de tekst hoge relevantie heeft voor de pagina (denk ik niet van toepassing).

Reply:
Als alternatief zou mijn voorstel zijn:
```
<h2>Workshops</h2>
<p><b>Indien je aan de slag wilt met een les module, dan raden wij je aan om de volgende workshops als basis te nemen:</b></p>
```